dataclients/kubernetes/definitions/routegroupvalidator.go

```diff
@@ -37,6 +38,7 @@ func (rgv *RouteGroupValidator) Validate(item *RouteGroupItem) error {
 	var errs []error
 	errs = append(errs, rgv.filtersValidation(item))
 	errs = append(errs, rgv.predicatesValidation(item))
+	errs = append(errs, rgv.validateBackends(item))
```

Comment:
Note how naming is not consistent here. IMO filtersValidation shoudl be validateFilters and predicatesValidation - validatePredicates.